.github/workflows/build-wheels.yml

```diff
@@ -29,11 +29,23 @@ jobs:
         with:
           python-version: "3.9"
 
+      - name: Download wheels from commit OpenAssetIO feature branch
```

Comment:
[question] So is the idea to keep the `drop!` commit in the history of the feature branch, then drop before merging feature to main?

Reply:
Seem reasonable? I was half thinking that well squash this feature branch down end the end - what do you recon?